core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java

```diff
@@ -60,7 +71,20 @@ public void createTable(
   @Override
   public void createNamespace(String namespace, Map<String, String> options)
       throws ExecutionException {
-    CreateKeyspace query = SchemaBuilder.createKeyspace(quoteIfNecessary(namespace));
+    try {
+      createKeyspace(namespace, options);
+      createMetadataKeyspaceIfNotExists();
+      createKeyspacesTableIfNotExists();
+      insertIntoKeyspacesTable(namespace);
+    } catch (IllegalArgumentException e) {
```

Comment:
I probably wanted to bubble up the IllegalArgumentException instead of an `ExecutionException` for a deliberate reason, but I can't remember nor find the reason again. 
Thanks, I will remove it as it does not seem important anyway.

Reply:
Thanks to a unit test I had written, I found out why I had this specific handling for the `IllegalArgumentException`. 
If the provided replication strategy in the creation options is not recognized, the underlying method `ReplicationStrategy.fromString()` will throw an `IllegalArgumentException`. 
Eventually, keeping it like this is better in my opinion. What do you think?

Reply:
If ExecutionException shouldn't be thrown when a provided replication strategy isn't recognized, we should leave it as is. IMO, we may want to remove this `catch (IllegalArgumentException e)` otherwise. But this discussion won't block this PR from being merged and it's up to you.

Reply:
Ok.

> If ExecutionException shouldn't be thrown when a provided replication strategy isn't recognized, we should leave it as is
Yes, I think it is more appropriate that an IllegalArgumentException is thrown from this method in that case.